extractor/src/test/java/org/schabi/newpipe/extractor/services/youtube/search/YoutubeSearchExtractorChannelOnlyTest.java

```diff
@@ -77,7 +92,7 @@ public void testChannelUrl() {
             if (item instanceof ChannelInfoItem) {
                 ChannelInfoItem channel = (ChannelInfoItem) item;
 
-                if (channel.getSubscriberCount() > 5e7) { // the real PewDiePie
+                if (channel.getSubscriberCount() > 5e7 && channel.getName().equals("PewDiePie")) { // the real PewDiePie
```

Comment:
Is that a good idea? Because we are talking about channel name, something youtubers (can) change often & easily. Check for id would be better (unchangeable)

Reply:
You're right.

Reply:
Then change 5e7 (50M) to 1e8, ie 100M

Reply:
The problem is T-Series showing up in the results, so that won't help. What would help is a `break` after the first channel is found that matches the subscriber criteria.